tests/nested/manual/minimal-smoke/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: execute smoke tests in a nested Ubuntu Core VM that meets the minimal requirements
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64, ubuntu-20.04-64]
+
+prepare: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    create_nested_core_vm
+
+execute: |
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+
+    # see https://docs.ubuntu.com/core/en/#advantages-for-iot for minimum
+    # requirements
+    if is_core_20_nested_system ; then
+        # TODO:UC20: due to https://bugs.launchpad.net/snapd/+bug/1878541 once fixed
+        # it should be 256 as well
+        MINIMAL_MEM=1536
+        NESTED_SPREAD_SYSTEM=ubuntu-core-20-64
+    elif is_core_18_nested_system; then
+        NESTED_SPREAD_SYSTEM=ubuntu-core-18-64
+        MINIMAL_MEM=256
+    elif is_core_16_nested_system; then
+        NESTED_SPREAD_SYSTEM=ubuntu-core-16-64
```

Comment:
shouldn't this var already be set?

Reply:
there's a SPREAD_SYSTEM, but it's set to whatever is the host executing the test (i.e. ubuntu-16.04-64 and so on).